### PR TITLE
feat(audit): RHICOMPL-1408 log report download

### PR DESCRIPTION
**Pull request by vkrizan**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [X] Input Validation
- [X] Output Encoding
- [X] Authentication and Password Management
- [X] Session Management
- [X] Access Control
- [X] Cryptographic Practices
- [X] Error Handling and Logging
- [X] Data Protection
- [X] Communication Security
- [X] System Configuration
- [X] Database Security
- [X] File Management
- [X] Memory Management
- [X] General Coding Practices

### CODE REVIEW

**Review by skateman — APPROVED** [review]
The Seal of Approval

File: app/services/safe_downloader.rb [hunk 1]
```diff
@@ -48,7 +48,12 @@ def report_contents(downloaded_file)
     end
 
     def open_url(url, options)
-      url.open(options)
+      urlopen = url.open(options)
+      Rails.logger.audit_success("Downloaded report from URL: #{url}")
+      urlopen
+    rescue *DOWNLOAD_ERRORS
+      Rails.logger.audit_fail("Failed to dowload report from URL: #{url}")
```
Comment (akofink) [inline app/services/safe_downloader.rb:55]: Typo: `s/dowload/download/`

Comment (vkrizan) [reply app/services/safe_downloader.rb:55]: :facepalm:

Comment (skateman) [reply app/services/safe_downloader.rb:55]: Haha, I missed it too :rofl:

Comment (vkrizan) [reply app/services/safe_downloader.rb:55]: Added the fix to #877

**Pull request merged**